src/Microsoft.OData.Client/BulkUpdateGraph.cs

```diff
@@ -0,0 +1,146 @@
+﻿//---------------------------------------------------------------------
+// <copyright file="BulkUpdateGraph.cs" company="Microsoft">
+//      Copyright (C) Microsoft Corporation. All rights reserved. See License.txt in the project root for license information.
+// </copyright>
+//---------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.OData.UriParser;
+
+namespace Microsoft.OData.Client
+{
+    /// <summary>
+    /// Contains information about descriptors and their related descriptors. 
+    /// </summary>
+    internal sealed class BulkUpdateGraph

```

Comment:
what's the meaning of "Graph"?

Reply:
It is used to represent the relationships between the various descriptors.